.spin/bin/env

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+# Advice on how to get the path of the current script is endless - this seems
+# to work for what we want on the systems we want
+SCRIPT_PATH=`dirname "$0"`
+SCRIPT_PATH=`eval "cd \"$SCRIPT_PATH\" && pwd"`
+TRUFFLERUBY_DIR=$SCRIPT_PATH/../..
+
 function jt {
-  ruby tool/jt.rb "$@"
+  ruby $TRUFFLERUBY_DIR/tool/jt.rb "$@"
```

Comment:
Sorry, GitHub lost my other comment. I said: it’s safest to use quotes here in case `$TRUFFLERUBY_DIR` happens to contain any spaces or glob characters:

```suggestion
  ruby "$TRUFFLERUBY_DIR/tool/jt.rb" "$@"
```

Reply:
Indeed, and shellcheck noticed both issues:
```
shellcheck .spin/bin/env

In .spin/bin/env line 5:
SCRIPT_PATH=`dirname "$0"`
            ^------------^ SC2006: Use $(...) notation instead of legacy backticked `...`.

Did you mean: 
SCRIPT_PATH=$(dirname "$0")


In .spin/bin/env line 6:
SCRIPT_PATH=`eval "cd \"$SCRIPT_PATH\" && pwd"`
            ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.

Did you mean: 
SCRIPT_PATH=$(eval "cd \"$SCRIPT_PATH\" && pwd")


In .spin/bin/env line 10:
  ruby $TRUFFLERUBY_DIR/tool/jt.rb "$@"
       ^--------------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
  ruby "$TRUFFLERUBY_DIR"/tool/jt.rb "$@"

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
  https://www.shellcheck.net/wiki/SC2006 -- Use $(...) notation instead of le...
```